index.js

```diff
@@ -83,6 +83,14 @@ const create = (win, options) => {
 					webContents(win).insertText(clipboardContent);
 				}
 			}),
+			copyImage: decorateMenuItem({
+				id: 'copyImage',
+				label: 'Copy Image',
+				visible: props.mediaType === 'image',
+				click() {
+					webContents(win).copyImageAt(props.x, props.y);
+				}
+			}),
```

Comment:
Not the correct order. The order here should follow the order shown in the menu.